### DISCUSSION TITLE
Can somebody explain how Mapping Config COMBO BASE values work?

### DISCUSSION THREAD

**Discussion by VideogameScrapbook** [darthcloud/BlueRetro#1107] [Q&A]
Can somebody explain how Mapping Config COMBO BASE values work? I tried Googling "COMBO BASE 1" which has only 1 result in another language without proper explanation.

**Comment by AaronZ101** [comment]
Did you figure this out? I found the same forum post you mentioned which is in PT-BR but reading what they said didn't make any sense and I couldn't view the images because the forum wanted me to sign in, I registered but it needs approval lol...

I'm basically wanting to have a mapping where I can hold one button (select) and then use the dpad for example to do left/right on the right stick, while also still being able to use Select with a regular press, not sure if blueretro can do this but with Steam mapping I can so it would be good if blueretro can do it too, but there's nothing in any of the guides even mentioning the "COMBO BASE" thing.

**Comment by VideogameScrapbook** [comment]
No, haven't figured it out.

**Comment by VideogameScrapbook** [comment]
@AaronZ101 , I was looking to do a similar thing as you described.

I did just now find this in the documentation, which seems to be about COMBO BASE values but the explanation could be improved:
https://github.com/darthcloud/BlueRetro/wiki#6---button-combinations-functions

If I understand correctly, the combo bases can just be used for shortcuts for things like reset in some cases.

**Comment by AaronZ101** [comment]
hmm okay, maybe can add this as a feature request? Basically if you're using a controller that lacks as many buttons as a console would usually have (for example wanting to use a Sega Saturn-like BT controller on a PS2). For me I'm doing a fun project where I'm trying to beat "modern games" on a NES Switch Controller, to be honest the blueretro gave me the idea in the first place, its been fun/challenging, however this is a slight roadblock where I need to use combinations, I realize the niche-ness of this but I think it would still be a good thing to implement.

On Steam, you can do this quite easily by making what it calls a "Hold Action Set Layer" which remaps buttons on a controller as a certain button is being held (I set it 200ms+), and then removes the "Layer" when the button is released, while simply "regular pressing" that same button will have its own action. I only used it to remap stuff on the Dpad to not cheese it too much. I know the blueretro has a thing for "Output 1", "Output 2", "Output 3" etc. but not sure how that works or if its just when multiple controllers are synced up?

https://imgur.com/a/WtkzcWb

**Comment by Sowden** [comment]
Not to bump this again.  But I don't quite understand the button combos either.  The documentation talks about button combos, but it doesn't make a lot of sense.  I'm wondering if this function is exclusive to hw2 (Lazer Bears GC Replacement Board) and not to hw1 (Retrotimes GC Receiver Adapter).  Has anyone here set this up and care to shed some light on how to do so?  Thanks.
